frontend/src/components/ResourceTr.scss

```diff
@@ -0,0 +1,5 @@
+.odh-resource-tr--deleting {
+  opacity: 0.5;
+  transition: var(--pf-global-Transition);
+  transition-property: opacity;
+}
```

Comment:
@kywalker-rh just for you I've added a transition on the opacity. This uses PFs default of 250ms transition with timing function ease-in-out. When the resource is first marked for deletion, the row will transition to 50% opacity. However once the resource is gone, the row will be removed immediately with no transition.

Reply:
I'll take it... thanks for adding it!